cmd/restic/cmd_prune.go

```diff
@@ -170,7 +171,18 @@ func runPruneWithRepo(opts PruneOptions, gopts GlobalOptions, repo *repository.R
 		return err
 	}
 
-	return prune(opts, gopts, repo, usedBlobs)
+	removePacksFirst, repackPacks, removePacks, keepBlobs, stats, err :=
```

Comment:
I'd prefer to move `removePacksFirst, repackPacks, removePacks, keepBlobs` into a struct. That avoids a lot of code churn when we need a new IDSet or similar.

Reply:
Nice idea! Is changed!